[2020-03-21 10:32:25,414][routes.py:20][INFO][12524] - {"api_name":"http://127.0.0.1:5000/index", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 10:32:25,555][routes.py:36][INFO][12524] - {"api_name": "http://127.0.0.1:5000/index", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 10:32:44,297][routes.py:20][INFO][13004] - {"api_name":"http://127.0.0.1:5000/search_satellite", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 10:32:45,621][routes.py:20][INFO][12900] - {"api_name":"http://127.0.0.1:5000/search_satellite", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 10:32:47,018][routes.py:36][INFO][12900] - {"api_name": "http://127.0.0.1:5000/search_satellite", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 10:33:02,076][routes.py:36][INFO][13004] - {"api_name": "http://127.0.0.1:5000/search_satellite", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 10:55:08,453][routes.py:20][INFO][8896] - {"api_name":"http://127.0.0.1:5000/index", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 10:55:08,476][routes.py:36][INFO][8896] - {"api_name": "http://127.0.0.1:5000/index", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 10:56:05,602][routes.py:20][INFO][9884] - {"api_name":"http://127.0.0.1:5000/search_satellite", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 10:56:07,398][routes.py:36][INFO][9884] - {"api_name": "http://127.0.0.1:5000/search_satellite", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 11:09:15,954][routes.py:20][INFO][11828] - {"api_name":"http://127.0.0.1:5000/index", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 11:09:15,997][routes.py:36][INFO][11828] - {"api_name": "http://127.0.0.1:5000/index", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 11:09:18,484][routes.py:20][INFO][11640] - {"api_name":"http://127.0.0.1:5000/search_satellite", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 11:09:19,799][routes.py:36][INFO][11640] - {"api_name": "http://127.0.0.1:5000/search_satellite", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:27:50,015][routes.py:20][INFO][10900] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:27:50,101][routes.py:36][INFO][10900] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:31:12,043][routes.py:20][INFO][11544] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:31:12,045][routes.py:36][INFO][11544] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:31:26,858][routes.py:20][INFO][772] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:31:26,859][routes.py:36][INFO][772] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:31:28,261][routes.py:20][INFO][9660] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:31:28,262][routes.py:36][INFO][9660] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:32:07,910][routes.py:20][INFO][12108] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:32:07,911][routes.py:36][INFO][12108] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:35:43,756][routes.py:20][INFO][4472] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:35:43,766][routes.py:36][INFO][4472] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:37:07,010][routes.py:20][INFO][12528] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:37:07,056][routes.py:36][INFO][12528] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:37:08,091][routes.py:20][INFO][9992] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:37:08,092][routes.py:36][INFO][9992] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:37:32,768][routes.py:20][INFO][3016] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:37:32,768][routes.py:36][INFO][3016] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:38:10,392][routes.py:20][INFO][656] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:38:10,394][routes.py:36][INFO][656] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:38:21,134][routes.py:20][INFO][12520] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:38:21,136][routes.py:36][INFO][12520] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:38:35,664][routes.py:20][INFO][5236] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:38:35,666][routes.py:36][INFO][5236] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:38:44,016][routes.py:20][INFO][11320] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:38:44,017][routes.py:36][INFO][11320] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:39:30,230][routes.py:20][INFO][7576] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:39:30,233][routes.py:36][INFO][7576] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:40:09,281][routes.py:20][INFO][9660] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:40:09,282][routes.py:36][INFO][9660] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:40:14,341][routes.py:20][INFO][4568] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:40:14,341][routes.py:36][INFO][4568] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 12:40:33,031][routes.py:20][INFO][8996] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 12:40:33,032][routes.py:36][INFO][8996] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 13:47:33,386][routes.py:20][INFO][3680] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 13:47:33,386][app.py:1891][ERROR][3680] - Exception on / [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 44, in index
    g.user = None
NameError: name 'g' is not defined
[2020-03-21 13:47:33,606][routes.py:33][INFO][3680] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 13:47:51,660][routes.py:20][INFO][12704] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 13:47:51,661][app.py:1891][ERROR][12704] - Exception on / [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 45, in index
    return redirect(url_for('auth.login'))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2215, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\helpers.py", line 358, in url_for
    endpoint, values, method=method, force_external=external
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\werkzeug\routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'auth.login'. Did you mean 'views.index' instead?
[2020-03-21 13:47:51,799][routes.py:33][INFO][12704] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 13:48:15,336][routes.py:20][INFO][10624] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 13:48:15,337][app.py:1891][ERROR][10624] - Exception on / [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 45, in index
    #     return redirect(url_for('auth.login'))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2215, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\helpers.py", line 358, in url_for
    endpoint, values, method=method, force_external=external
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\werkzeug\routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'auth.login'. Did you mean 'views.index' instead?
[2020-03-21 13:48:15,341][routes.py:33][INFO][10624] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 13:48:21,223][routes.py:20][INFO][13012] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 13:48:21,234][routes.py:36][INFO][13012] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 13:49:16,524][routes.py:20][INFO][11208] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 13:49:16,525][routes.py:36][INFO][11208] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 13:50:16,684][routes.py:20][INFO][12460] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 13:50:16,685][routes.py:36][INFO][12460] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 13:50:24,169][routes.py:20][INFO][4884] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 13:50:24,169][routes.py:36][INFO][4884] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 13:50:47,892][routes.py:20][INFO][3928] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 13:50:47,892][routes.py:36][INFO][3928] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 13:51:09,332][routes.py:20][INFO][4856] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 13:51:09,333][routes.py:36][INFO][4856] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 13:51:16,381][routes.py:20][INFO][8216] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 13:51:16,382][routes.py:36][INFO][8216] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 13:54:52,265][routes.py:20][INFO][9400] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 13:54:52,266][routes.py:36][INFO][9400] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 13:55:49,214][routes.py:20][INFO][9992] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 13:55:49,215][routes.py:36][INFO][9992] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:00:04,926][routes.py:20][INFO][12976] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:00:04,989][app.py:1891][ERROR][12976] - Exception on /login [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 98, in login
    return render_template('login.html', title="Login")
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 140, in render_template
    ctx.app,
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\templates\login.html", line 24, in top-level template code
    <a id="login_a" style="font-weight: bold;" href="{{ url_for('auth.login') }}">Login</a> |
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2215, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\helpers.py", line 358, in url_for
    endpoint, values, method=method, force_external=external
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\werkzeug\routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'auth.login'. Did you mean 'views.login' instead?
[2020-03-21 19:00:05,080][routes.py:33][INFO][12976] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 19:02:27,977][routes.py:20][INFO][312] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:02:27,987][app.py:1891][ERROR][312] - Exception on /login [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 98, in login
    return render_template('login.html', title="Login")
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 140, in render_template
    ctx.app,
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\templates\login.html", line 47, in top-level template code
    <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
jinja2.exceptions.UndefinedError: 'csrf_token' is undefined
[2020-03-21 19:02:27,991][routes.py:33][INFO][312] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 19:03:26,903][routes.py:20][INFO][5312] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:03:26,912][app.py:1891][ERROR][5312] - Exception on /login [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 98, in login
    return render_template('login.html', title="Login")
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 140, in render_template
    ctx.app,
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\templates\login.html", line 47, in top-level template code
    <!--<input type="hidden" name="csrf_token" value="{{ csrf_token() }}">-->
jinja2.exceptions.UndefinedError: 'csrf_token' is undefined
[2020-03-21 19:03:26,916][routes.py:33][INFO][5312] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 19:30:51,383][routes.py:20][INFO][8752] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:30:51,508][app.py:1891][ERROR][8752] - Exception on /login [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 98, in login
    return render_template('login.html', title="Login")
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 140, in render_template
    ctx.app,
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\templates\login.html", line 47, in top-level template code
    <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_wtf\csrf.py", line 35, in generate_csrf
    message='A secret key is required to use CSRF.'
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_wtf\csrf.py", line 128, in _get_config
    raise RuntimeError(message)
RuntimeError: A secret key is required to use CSRF.
[2020-03-21 19:30:51,531][routes.py:33][INFO][8752] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 19:32:27,662][routes.py:20][INFO][11088] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:32:27,693][routes.py:36][INFO][11088] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:34:09,474][routes.py:20][INFO][7496] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:34:09,475][routes.py:36][INFO][7496] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:35:34,128][routes.py:20][INFO][7608] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:35:34,139][routes.py:36][INFO][7608] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:36:58,034][routes.py:20][INFO][792] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:36:58,041][routes.py:36][INFO][792] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:37:07,004][routes.py:20][INFO][8868] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:37:07,005][routes.py:36][INFO][8868] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:37:07,730][routes.py:20][INFO][8312] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:37:07,731][routes.py:36][INFO][8312] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:38:04,064][routes.py:20][INFO][1596] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:38:04,069][routes.py:36][INFO][1596] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:41:11,189][routes.py:20][INFO][128] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:41:11,195][routes.py:36][INFO][128] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:41:12,106][routes.py:20][INFO][6024] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:41:12,106][routes.py:36][INFO][6024] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:44:24,428][routes.py:20][INFO][4760] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:44:24,430][app.py:1891][ERROR][4760] - Exception on / [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 46, in index
    return render_template('index.html', title = "Index")
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\loaders.py", line 117, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
[2020-03-21 19:44:24,497][routes.py:33][INFO][4760] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 19:52:20,430][routes.py:20][INFO][10484] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:52:20,432][app.py:1891][ERROR][10484] - Exception on / [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 46, in index
    return render_template('../templates/index.html', title = "Index")
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\loaders.py", line 117, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /templates/index.html
[2020-03-21 19:52:20,438][routes.py:33][INFO][10484] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 19:54:22,715][routes.py:20][INFO][9928] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:54:22,715][app.py:1891][ERROR][9928] - Exception on / [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 46, in index
    return render_template('../templates/index.html', title = "Index")
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\loaders.py", line 117, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: ../templates/index.html
[2020-03-21 19:54:22,725][routes.py:33][INFO][9928] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 19:54:58,294][routes.py:20][INFO][3432] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:54:58,318][routes.py:36][INFO][3432] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:55:48,795][routes.py:20][INFO][7484] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:55:48,803][routes.py:36][INFO][7484] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:55:50,127][routes.py:20][INFO][12320] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:55:50,128][routes.py:36][INFO][12320] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 19:59:52,273][routes.py:20][INFO][10212] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 19:59:52,279][routes.py:36][INFO][10212] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:01:34,520][routes.py:20][INFO][2820] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:01:34,528][routes.py:36][INFO][2820] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:01:35,542][routes.py:20][INFO][12020] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:01:35,544][routes.py:36][INFO][12020] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:02:16,896][routes.py:20][INFO][3380] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:02:16,922][routes.py:36][INFO][3380] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:04:14,914][routes.py:20][INFO][10832] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:04:14,925][routes.py:36][INFO][10832] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:04:15,704][routes.py:20][INFO][5744] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:04:15,705][routes.py:36][INFO][5744] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:07:36,538][routes.py:20][INFO][2868] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:07:36,561][routes.py:36][INFO][2868] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:08:29,169][routes.py:20][INFO][5780] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:08:29,169][routes.py:36][INFO][5780] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:08:34,091][routes.py:20][INFO][3492] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:08:34,092][routes.py:36][INFO][3492] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:08:46,554][routes.py:20][INFO][2108] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:08:46,559][routes.py:36][INFO][2108] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:08:56,548][routes.py:20][INFO][7352] - {"api_name":"http://127.0.0.1:5000/register", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:08:56,574][routes.py:36][INFO][7352] - {"api_name": "http://127.0.0.1:5000/register", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:09:42,886][routes.py:20][INFO][11436] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:09:42,887][routes.py:36][INFO][11436] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:11:38,780][routes.py:20][INFO][10564] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:11:38,782][app.py:1891][ERROR][10564] - Exception on / [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 46, in index
    return render_template('index.html', title = "Index")
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\loaders.py", line 117, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
[2020-03-21 20:11:38,813][routes.py:33][INFO][10564] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 20:11:58,325][routes.py:20][INFO][11344] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:11:58,327][app.py:1891][ERROR][11344] - Exception on / [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 46, in index
    return render_template('index.html', title = "Index")
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\loaders.py", line 117, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
[2020-03-21 20:11:58,330][routes.py:33][INFO][11344] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 20:12:19,157][routes.py:20][INFO][5832] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:12:19,159][app.py:1891][ERROR][5832] - Exception on / [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 46, in index
    return render_template('index.html', title = "Index")
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\loaders.py", line 117, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
[2020-03-21 20:12:19,169][routes.py:33][INFO][5832] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 20:12:20,902][routes.py:20][INFO][588] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:12:20,904][app.py:1891][ERROR][588] - Exception on / [GET]
Traceback (most recent call last):
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask_cors\extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\TedZhang\Courses\COMP8715\SAT-Track\server\views\routes.py", line 46, in index
    return render_template('index.html', title = "Index")
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\jinja2\loaders.py", line 117, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "D:\ProgramData\Miniconda3\envs\comp8715\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
[2020-03-21 20:12:20,906][routes.py:33][INFO][588] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"500"}
[2020-03-21 20:12:59,833][routes.py:20][INFO][12812] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:12:59,845][routes.py:36][INFO][12812] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:13:13,733][routes.py:20][INFO][10300] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:13:13,740][routes.py:36][INFO][10300] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:13:32,578][routes.py:20][INFO][12672] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:13:32,586][routes.py:36][INFO][12672] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:14:07,599][routes.py:20][INFO][11332] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:14:07,607][routes.py:36][INFO][11332] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:14:09,339][routes.py:20][INFO][8752] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:14:09,340][routes.py:36][INFO][8752] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:14:26,409][routes.py:20][INFO][10668] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:14:26,423][routes.py:36][INFO][10668] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:15:23,033][routes.py:20][INFO][10348] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:15:23,041][routes.py:36][INFO][10348] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:15:23,960][routes.py:20][INFO][10552] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:15:23,961][routes.py:36][INFO][10552] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:15:42,030][routes.py:20][INFO][13304] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:15:42,036][routes.py:36][INFO][13304] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:15:42,730][routes.py:20][INFO][6772] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:15:42,731][routes.py:36][INFO][6772] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:16:13,862][routes.py:20][INFO][4232] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:16:13,879][routes.py:36][INFO][4232] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:18:00,631][routes.py:20][INFO][12240] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:18:00,636][routes.py:36][INFO][12240] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:18:01,262][routes.py:20][INFO][12836] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:18:01,262][routes.py:36][INFO][12836] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:18:01,661][routes.py:20][INFO][12044] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:18:01,662][routes.py:36][INFO][12044] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:18:01,830][routes.py:20][INFO][10568] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:18:01,830][routes.py:36][INFO][10568] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:18:01,999][routes.py:20][INFO][3624] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:18:02,000][routes.py:36][INFO][3624] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:18:02,150][routes.py:20][INFO][13020] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:18:02,151][routes.py:36][INFO][13020] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:18:02,279][routes.py:20][INFO][5212] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:18:02,280][routes.py:36][INFO][5212] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:21:43,903][routes.py:20][INFO][5148] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:21:43,914][routes.py:36][INFO][5148] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:26:09,930][routes.py:20][INFO][6548] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:26:09,973][routes.py:36][INFO][6548] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:26:13,353][routes.py:20][INFO][3720] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:26:13,354][routes.py:36][INFO][3720] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:26:30,675][routes.py:20][INFO][3896] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:26:30,685][routes.py:36][INFO][3896] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:26:32,573][routes.py:20][INFO][9152] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:26:32,575][routes.py:36][INFO][9152] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:26:59,189][routes.py:20][INFO][7468] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:26:59,199][routes.py:36][INFO][7468] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:27:11,185][routes.py:20][INFO][1668] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:27:11,195][routes.py:36][INFO][1668] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:27:11,963][routes.py:20][INFO][12416] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:27:11,964][routes.py:36][INFO][12416] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:29:12,719][routes.py:20][INFO][13252] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:29:12,729][routes.py:36][INFO][13252] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:30:00,843][routes.py:20][INFO][1540] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:30:00,852][routes.py:36][INFO][1540] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:30:42,733][routes.py:20][INFO][10556] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:30:42,742][routes.py:36][INFO][10556] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:30:43,423][routes.py:20][INFO][1540] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:30:43,424][routes.py:36][INFO][1540] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:31:23,866][routes.py:20][INFO][10556] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:31:23,874][routes.py:36][INFO][10556] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:31:42,092][routes.py:20][INFO][1204] - {"api_name":"http://127.0.0.1:5000/register", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:31:42,098][routes.py:36][INFO][1204] - {"api_name": "http://127.0.0.1:5000/register", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:32:17,350][routes.py:20][INFO][10252] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:32:17,350][routes.py:36][INFO][10252] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:32:24,191][routes.py:20][INFO][13432] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:32:24,192][routes.py:36][INFO][13432] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:32:25,362][routes.py:20][INFO][13428] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:32:25,364][routes.py:36][INFO][13428] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:34,527][routes.py:20][INFO][12068] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:34,714][routes.py:36][INFO][12068] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:36,965][routes.py:20][INFO][4972] - {"api_name":"http://127.0.0.1:5000/register", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:36,970][routes.py:36][INFO][4972] - {"api_name": "http://127.0.0.1:5000/register", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:38,251][routes.py:20][INFO][13460] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:38,252][routes.py:36][INFO][13460] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:39,344][routes.py:20][INFO][3772] - {"api_name":"http://127.0.0.1:5000/register", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:39,345][routes.py:36][INFO][3772] - {"api_name": "http://127.0.0.1:5000/register", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:40,457][routes.py:20][INFO][12436] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:40,458][routes.py:36][INFO][12436] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:41,484][routes.py:20][INFO][13736] - {"api_name":"http://127.0.0.1:5000/register", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:41,485][routes.py:36][INFO][13736] - {"api_name": "http://127.0.0.1:5000/register", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:42,438][routes.py:20][INFO][13664] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:42,440][routes.py:36][INFO][13664] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:42,928][routes.py:20][INFO][11284] - {"api_name":"http://127.0.0.1:5000/register", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:42,930][routes.py:36][INFO][11284] - {"api_name": "http://127.0.0.1:5000/register", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:43,264][routes.py:20][INFO][4524] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:43,265][routes.py:36][INFO][4524] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:44,150][routes.py:20][INFO][12284] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:44,151][routes.py:36][INFO][12284] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:44,869][routes.py:20][INFO][12340] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:44,871][routes.py:36][INFO][12340] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:46,765][routes.py:20][INFO][8332] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:46,767][routes.py:36][INFO][8332] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:35:47,896][routes.py:20][INFO][12464] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:35:47,897][routes.py:36][INFO][12464] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:41:40,713][routes.py:84][INFO][12524] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:41:40,730][routes.py:100][INFO][12524] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:42:25,591][routes.py:84][INFO][9276] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:42:25,606][routes.py:100][INFO][9276] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:44:47,420][routes.py:81][INFO][12412] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:44:47,446][routes.py:97][INFO][12412] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:45:10,753][routes.py:81][INFO][5212] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:45:10,762][routes.py:97][INFO][5212] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:45:36,789][routes.py:81][INFO][1608] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:45:36,858][routes.py:97][INFO][1608] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 20:58:15,788][routes.py:81][INFO][4164] - {"api_name":"http://127.0.0.1:8000/index.html", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:58:15,790][routes.py:94][INFO][4164] - {"api_name": "http://127.0.0.1:8000/index.html", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"404"}
[2020-03-21 20:58:15,973][routes.py:81][INFO][9364] - {"api_name":"http://127.0.0.1:8000/favicon.ico", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:58:15,974][routes.py:94][INFO][9364] - {"api_name": "http://127.0.0.1:8000/favicon.ico", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"404"}
[2020-03-21 20:58:23,354][routes.py:81][INFO][13012] - {"api_name":"http://127.0.0.1:8000/index", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 20:58:23,356][routes.py:94][INFO][13012] - {"api_name": "http://127.0.0.1:8000/index", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"404"}
[2020-03-21 21:00:49,305][routes.py:81][INFO][2336] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:00:49,324][routes.py:97][INFO][2336] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:02:32,184][routes.py:81][INFO][3588] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:02:32,198][routes.py:97][INFO][3588] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:02:35,150][routes.py:81][INFO][5052] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:02:35,151][routes.py:97][INFO][5052] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:03:23,308][routes.py:81][INFO][13324] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:03:23,319][routes.py:97][INFO][13324] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:03:24,815][routes.py:81][INFO][12912] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:03:24,816][routes.py:97][INFO][12912] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:06:31,292][routes.py:81][INFO][12788] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:06:31,302][routes.py:97][INFO][12788] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:06:54,195][routes.py:81][INFO][5560] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:06:54,204][routes.py:97][INFO][5560] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:07:00,878][routes.py:81][INFO][14272] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:07:01,269][routes.py:97][INFO][14272] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:07:52,708][routes.py:81][INFO][13164] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:07:52,723][routes.py:97][INFO][13164] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:08:12,692][routes.py:81][INFO][7400] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:08:12,704][routes.py:97][INFO][7400] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:08:30,013][routes.py:81][INFO][5220] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:08:30,019][routes.py:97][INFO][5220] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:08:30,778][routes.py:81][INFO][9908] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:08:30,779][routes.py:97][INFO][9908] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:08:53,060][routes.py:81][INFO][7696] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:08:53,066][routes.py:97][INFO][7696] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:08:53,718][routes.py:81][INFO][4032] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:08:53,719][routes.py:97][INFO][4032] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:31:11,395][routes.py:81][INFO][13628] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:31:11,419][routes.py:97][INFO][13628] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:33:07,499][routes.py:85][INFO][13056] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:33:07,509][routes.py:101][INFO][13056] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:33:14,491][routes.py:85][INFO][13496] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:33:14,507][routes.py:101][INFO][13496] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:36:58,034][routes.py:81][INFO][13792] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:36:58,050][routes.py:97][INFO][13792] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:37:01,333][routes.py:81][INFO][4368] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:37:01,334][routes.py:97][INFO][4368] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:37:03,662][routes.py:81][INFO][1204] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:37:03,663][routes.py:97][INFO][1204] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:37:07,015][routes.py:81][INFO][11400] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:37:07,117][routes.py:97][INFO][11400] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:37:10,251][routes.py:81][INFO][10396] - {"api_name":"http://127.0.0.1:5000/register", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:37:10,274][routes.py:97][INFO][10396] - {"api_name": "http://127.0.0.1:5000/register", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:37:11,283][routes.py:81][INFO][1492] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:37:11,284][routes.py:97][INFO][1492] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:37:12,953][routes.py:81][INFO][14112] - {"api_name":"http://127.0.0.1:5000/register", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:37:12,955][routes.py:97][INFO][14112] - {"api_name": "http://127.0.0.1:5000/register", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:37:13,614][routes.py:81][INFO][14052] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:37:13,614][routes.py:97][INFO][14052] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:37:15,328][routes.py:81][INFO][13980] - {"api_name":"http://127.0.0.1:5000/register", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:37:15,329][routes.py:97][INFO][13980] - {"api_name": "http://127.0.0.1:5000/register", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:37:15,658][routes.py:81][INFO][13536] - {"api_name":"http://127.0.0.1:5000/register", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:37:15,659][routes.py:97][INFO][13536] - {"api_name": "http://127.0.0.1:5000/register", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:37:16,429][routes.py:81][INFO][10952] - {"api_name":"http://127.0.0.1:5000/login", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:37:16,430][routes.py:97][INFO][10952] - {"api_name": "http://127.0.0.1:5000/login", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:39:13,665][routes.py:81][INFO][12896] - {"api_name":"http://127.0.0.1:5000/", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:39:13,675][routes.py:97][INFO][12896] - {"api_name": "http://127.0.0.1:5000/", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
[2020-03-21 21:39:22,180][routes.py:81][INFO][4032] - {"api_name":"http://127.0.0.1:5000/register", "user_ip":"127.0.0.1", "user_name":"None"}
[2020-03-21 21:39:22,191][routes.py:97][INFO][4032] - {"api_name": "http://127.0.0.1:5000/register", "user_ip": "127.0.0.1", "user_name": "None", "status_code":"200"}
